include/mbedtls/ssl.h

```diff
@@ -2424,6 +2467,32 @@ typedef int mbedtls_ssl_export_keys_ext_t( void *p_expkey,
                                            const unsigned char client_random[32],
                                            const unsigned char server_random[32],
                                            mbedtls_tls_prf_types tls_prf_type );
+#endif /* MBEDTLS_SSL_PROTO_TLS1 || MBEDTLS_SSL_PROTO_TLS1_1 || \
+          MBEDTLS_SSL_PROTO_TLS1_2 */
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+/**
+ * \brief           Callback type: Export TLS 1.3 secrets (client early traffic
+ *                  secret, client handshake traffic secret, server handshake
+ *                  traffic secret, client application 0 traffic secret, server
+ *                  application 0 traffic secret, exporter master secret)
+ *
+ * \note            This is required for certain uses of TLS, e.g. EAP-TLS
```

Comment:
I don't think this comment applies to TLS 1.3, does it? Those applications would use the exporter secret from TLS 1.3.

Reply:
I will have to check again with EAP-TLS 1.3 whether there is any change needed here. Will do that later.